.eslintrc.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable import/no-commonjs */
+module.exports = {
```

Comment:
was this copied from wonder-stuff or somewhere? Could you indicate that in the PR description?

Reply:
I think we're probably want to copy a bunch of things over from webapp and/or wonder-blocks since wonder-stuff doesn't use React. I'm going to do that in a follow up PR.